### PR TITLE
add highlights for azure spring boot

### PR DESCRIPTION
**Pull request by yiliuTo**

### CODE REVIEW

File: releases/2020-08/index.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ sidebar: releases_sidebar
 * Azure Cognitive Search SDK for .NET added `FieldBuilder` to help easily build a search index from a model type.
 * Added ObjectSerializer and JsonSerializer APIs to support pluggable serialization within SDKs.
 * Added support for Key Vault service version 7.1 for Keys, Secrets, and Certificates
+* Java Message Service (JMS) 2.0 preview for Azure Service Bus premium tier, which enables more lift and shift scenarios and significantly increased JMS API compatibility.
```
Comment (jongio) [inline releases/2020-08/index.md:14]: Are these Azure SDK related?  Can you point to these PRs so I can research?  Thanks, Jon

Comment (yiliuTo) [reply releases/2020-08/index.md:14]: Hi, the related PRs are as follows:

1. JMS: the mentioned update to support Azure Service Bus premium tier via JMS 2.0 is imported in our underlying library "azure-servicebus-jms:0.0.4", and in our azure-spring-boot module, we update the underlying version to `0.0.4`, here is the [PR](https://github.com/Azure/azure-sdk-for-java/pull/13854).
2. Key Vault starter: due to our spring repo was just migrated into SDK repo, so some of the mentioned features are done in the old repo while hadn't been released until Aug release.
   - case sensitive name support: https://github.com/microsoft/azure-spring-boot/pull/885
   - multiple KeyVault support: https://github.com/microsoft/azure-spring-boot/pull/880
   - actuator support & refreshing logic update: https://github.com/Azure/azure-sdk-for-java/pull/12318
3. Spring Boot: removed this entry.

Comment (jongio) [reply releases/2020-08/index.md:14]: @joshfree - Can you please comment on what you want in Track 2 release notes.  Thanks, Jon

**Comment by jongio** [comment]
@samvaity - Can you please review?  Thanks, Jon

File: releases/2020-08/index.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ sidebar: releases_sidebar
...
+* New features added to the Key Vault stater to support case sensitive keys, connect to multiple Key Vaults from a single app, Spring Boot Actuator and performance improvements when keys are refreshed.
```
[the remark this answers is not in the document]
Comment (joshfree) [reply releases/2020-08/index.md:15]: Since this is a language-agnostic announcement, could this be updated to more clearly state it's Java-specific for readers who are not familiar with spring?

File: releases/2020-08/index.md (hunk 1 above)
Comment (hemanttanwar) [reply releases/2020-08/index.md:14]: @jongio  I will look into it today.

Comment (hemanttanwar) [reply releases/2020-08/index.md:14]: I spoke to @jialindai  and  he will put additional detail about track 2 libraries in here.

File: releases/2020-08/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yiliuTo) [reply releases/2020-08/index.md]: Hi @joshfree , I have updated Spring content to try to highlight its Java-specific property and also added details about track 2 underpinning libraries. Thanks.

File: releases/2020-08/index.md [hunk 3]
```diff
@@ -11,9 +11,10 @@ sidebar: releases_sidebar
 * Azure Cognitive Search SDK for .NET added `FieldBuilder` to help easily build a search index from a model type.
 * Added ObjectSerializer and JsonSerializer APIs to support pluggable serialization within SDKs.
 * Added support for Key Vault service version 7.1 for Keys, Secrets, and Certificates
-* Java Message Service (JMS) 2.0 preview for Azure Service Bus premium tier, which enables more lift and shift scenarios and significantly increased JMS API compatibility.
-* New features added to the Key Vault stater to support case sensitive keys, connect to multiple Key Vaults from a single app, Spring Boot Actuator and performance improvements when keys are refreshed.
-* Azure Spring Boot updated to Spring Boot 2.3.2.
+* [Java Message Service (JMS) 2.0 preview for Azure Service Bus premium tier](https://azure.microsoft.com/blog/announcing-preview-of-java-message-service-2-over-amqp-on-azure-service-bus/), which enables more lift and shift scenarios and significantly increased JMS API compatibility.
```
Comment (jongio) [inline releases/2020-08/index.md:14]: I don't think this one is SDK related

File: releases/2020-08/index.md [hunk 4]
```diff
@@ -11,9 +11,10 @@ sidebar: releases_sidebar
...
+* New features added to Java Spring Boot starter of Key Vault to support case sensitive keys, connect to multiple Key Vaults from a single app, Spring Boot Actuator and performance improvements when keys are refreshed.
```
Comment (jongio) [inline releases/2020-08/index.md:15]: Is there a link they can go to find more info?

File: releases/2020-08/index.md [hunk 5]
```diff
@@ -11,9 +11,10 @@ sidebar: releases_sidebar
...
+* Upgraded underpinning Java Key Vault SDK for Azure Spring starters from track1 to track2 libraries for better performance, consistency, and compatibility.
```
Comment (jongio) [inline releases/2020-08/index.md:16]:
```suggestion
* Upgraded underpinning Java Key Vault SDK for Azure Spring starte libraries for better performance, consistency, and compatibility.
```

File: releases/2020-08/index.md [hunk 6]
```diff
@@ -11,9 +11,10 @@ sidebar: releases_sidebar
...
+* Azure Spring Boot updated to Spring Boot 2.3.2, which addresses many bug fixes and cleans up all warnings at build time.
```
Comment (jongio) [inline releases/2020-08/index.md:17]: Is is this SDK related?

File: releases/2020-08/index.md (hunk 3 above)
Comment (jongio) [inline releases/2020-08/index.md:14]: If we end up with more than one Java Spring item here, please put them under a "Java Spring Updates" bullet - only do that if we have more than one. Thanks

File: releases/2020-08/index.md (hunk 6 above)
Comment (yiliuTo) [reply releases/2020-08/index.md]: Hi @jongio , thanks for the comments. For both the first entry and fourth entry are not directly SDK related, but the JMS 2.0 and Spring Boot 2.3.3 are parts of the important underpinning libraries of our Spring projects, especially for our starters which can be understood as synthesis of dependencies. So updates of them enhance the Spring project, and in this way, do you think it is suitable to put these two entries here? If not then I will remove them. Thanks again.

Comment (jongio) [reply releases/2020-08/index.md]: How about you expand the highlight to include why this is related to the new SDK?  That would be helpful.

Comment (seanli1988) [reply releases/2020-08/index.md]: @jongio I responded your question in an email.

**Comment by jongio** [comment]
@samvaity - Is this ready to be merged?
